### PR TITLE
Update campsites and code-of-conduct links

### PR DESCRIPTION
**Pull request by lilyheart**

### CODE REVIEW

**Comment by Rafase282** [comment]
Thank you.

File: Official-Free-Code-Camp-Chat-Rooms.md [hunk 1]
```diff
@@ -29,6 +29,6 @@ The following are our official chat rooms.
 | [CurriculumDevelopment](https://gitter.im/freecodecamp/CurriculumDevelopment) | help us improve our open source curriculum |
 | [DataScience](https://gitter.im/freecodecamp/DataScience) | help us understand our gigs and gigs of public data |
 
-By joining these chat rooms, you accept our [Code of Conduct](https://github.com/FreeCodeCamp/freecodecamp/wiki/Code-of-Conduct).
+By joining these chat rooms, you accept our [Code of Conduct](Code-of-Conduct).
```
Comment (abhisekp) [inline Official-Free-Code-Camp-Chat-Rooms.md]: @Rafase282 shouldn't we link to https://freecodecamp.com/code-of-conduct page here?

Comment (lilyheart) [inline Official-Free-Code-Camp-Chat-Rooms.md]: Do we need two separate code of conduct pages?  They are not 100% identical.

Comment (Rafase282) [inline Official-Free-Code-Camp-Chat-Rooms.md]: We would have to check with the core team on this. They created both of them, so will they keep it in sync, should we keep it in sync or just use the one on the site?

Comment (theflametrooper) [inline Official-Free-Code-Camp-Chat-Rooms.md]: I think we should just keep the one on the site to avoid synchronisation issues.

**Pull request merged**